### PR TITLE
fix(pie-label):#16208 When the total number of data is less than or equal to 2, the first data label shows a bug

### PR DESCRIPTION
**Pull request by 116050423**
## Brief Information

This pull request is in the type of:

- [x] bug fixing
- [ ] new feature
- [ ] others

### Fixed issues

- #16208 

## Misc

- [ ] The API has been changed (apache/echarts-doc#xxx).
- [ ] This PR depends on ZRender changes (ecomfe/zrender#xxx).

### Related test cases or examples to use the new APIs

N.A.

### Merging options

- [x] Please squash the commits into a single one when merging.

### CODE REVIEW

**Comment by 116050423** [comment]
@pissang @Ovilia Hi, I passed the local test + visual test, would you please help to review my code, thanks.

**Comment by susiwen8** [comment]
Could you add a test case for it?

**Comment by 116050423** [comment]
> 

test case added, please help to review, thanks

File: test/pie-label.html [hunk 1]
```diff
@@ -770,5 +771,37 @@
...
+                            ],
+                            label: {
+                                show: true
+                            },
+                        }
+                    ]
+                };
+
+                var chart = testHelper.create(echarts, 'main9', {
+                    title: 'the first label shoud be rendered correctly when it\'s followed by several zero values',
```
Comment (pissang) [inline test/pie-label.html:799]: You can reference this issue in the tittle so we can trace the origin of this case.

Comment (116050423) [reply test/pie-label.html]: @pissang It's done, please review again, thanks.

**Review by pissang — APPROVED** [review]

**Pull request merged**